src/server.js

```diff
@@ -8,13 +8,20 @@ var express = require('express');        // call express
 var app = express();                 // define our app using express
 var bodyParser = require('body-parser');
 const request = require('request-promise');
+var urlencodedParser = bodyParser.urlencoded({ limit: '50mb', extended: true });
 var testHarnessPath = "./src/app";
 var port = process.env.PORT || 8080;        // set our port
+// define our ticket parameters
+var testerName
```

Comment:
I would not have these variables, specifically testerName, testerEnv etc as global variables. Instead, define them in their instance of use. That means in the body of the receiveData route, define them there. I can work with you to clean up that and do it in a Promise way.

Reply:
ok will do